openquake/hazardlib/tests/expected_sources.toml

```diff
@@ -134,7 +134,13 @@ _downDip = 0.75
 _weight = 0.7
 
 [simpleFaultSource.slipList]
-slip = [ "90.0", "135.0",]
+[[simpleFaultSource.slipList.slip]]
+_weight = 0.7
+text = "90.0"
+
+[[simpleFaultSource.slipList.slip]]
+_weight = 0.3
+text = "135.0"
 
```

Comment:
I would have expected text to be a float, not a string. If you see below the coordinates are converted into floats indeed. Idem for the weight.

Reply:
Done